packages/input-edit/src/components/InputEditSearch/InputEditSearch.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import SimpleFormApp from 'tocco-simple-form/src/main'
+import _debounce from 'lodash/debounce'
+import {tqlBuilder} from 'tocco-util'
+import PropTypes from 'prop-types'
+
+const InputEditSearch = ({form, model, setSearchFields}) => {
+  if (form.children && model.paths) {
+    return <SimpleFormApp form={form}
+      model={model}
+      onChange={_debounce(handleChange(form, setSearchFields), 500)}
+      noButtons={true}/>
+  } else {
+    return null
+  }
+}
+
+const handleChange = (form, setSearchFields) => ({values, valid}) => {
+  if (valid && Object.keys(values).length > 0) {
+    const tql = Object.entries(values)
+      .map(([path, value]) => {
+        const parsedPath = path.replace('--', '.')
```

Comment:
little background here: we use redux-forms to handle the form state. Redux-forms has a limitation that field ids should not contain any dots in favour of nested fields. So as a workaround we replace any dots with two dashes. And before we send the values to the backend we need to rename them back.

But i dont think simple-form should  call onChange with the renamed keys. Hence we should rename the field ids in simple form.
Somewhere here:
https://github.com/tocco/tocco-client/blob/7076bc1f7024a74dfbd96409b87cbea4613127d6/packages/simple-form/src/modules/simpleForm/sagas.js#L31